src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
```

Comment:
Keep your attributes correctly formatted. If the HTML element has more than 2 attributes, start each one, including the first, on a new line with 2-space indentation related to the tag.